### DISCUSSION TITLE
building cleaning pipeline

### DISCUSSION THREAD

**Discussion by remic33** [EtienneAb3d/WhisperHallu#11] [Ideas]
Hey,
I am looking at your code to see if i could build a pipeline to clean audio data before feeding it to whisper. Maybe it's something you should consider to bring more usage, and do not have too much depencies with whisper evolutions.

**Comment by EtienneAb3d** [comment]
@remic33 
Not sure to understand your remark. There is only 2 versions of Whisper:
- Standard Wisper, because it's quite mandatory
- Faster Whisper, because it brings a faster solution with less GPU RAM usage

What would you like me to remove?
